File: entregavel1_javascript/js/f_contagem.js
function count(n, conjuntoNumeros) {
    // let n = prompt("Digite um número e veja quantos valores inteiros existem entre 1 (inclusive) e o num. que vc digitou:");

    // let array = ((prompt(`Agora escreva os ${n} números separados por virgula ( Ex.: 1,2,3.5,4 )`)).split(","));
    let array = conjuntoNumeros.split(",");

    let totalOfNumbers = 0;

    for (let index = 0; index < array.length; index++) {
        if(isInterger(array[index], n)) {
            totalOfNumbers++;
        }
    }

   // alert(`Total de números inteiros: ${totalOfNumbers}`);

   return `Total de números inteiros: ${totalOfNumbers}`;
}

function isInterger(strValue = "", n = "") {
    if(!strValue.includes(".")){
        if(parseInt(strValue)<=n) {
            return true;
        } else {
            return false;
        }
    } else {
        let decimalValue = strValue.split("\\.");

        if(parseInt(decimalValue[0])<=n && parseInt(decimalValue[1])==0) {
            return true;
        } else {
            return false;
        }
    }
}

module.exports = { count };

File: entregavel1_java/classes/F_Contagem.java
package entregavel1_java.classes;

import java.util.Scanner;

public class F_Contagem {

    public static String count() {
        Scanner sc1 = new Scanner(System.in);
        String arrayStr = "";

        System.out.println("Digite um número e veja quantos valores inteiros existem entre 1 (inclusive) e o num. que vc digitou:");
        int n = sc1.nextInt();

        for (int i = 0; i < n; i++) {
            System.out.println("Informe o "+(i+1)+"° num (de "+n+"): ");
            String aux = sc1.next();
            arrayStr = arrayStr + aux;
            if(i!=n-1) {
                arrayStr = arrayStr + ",";
            }
        }
        sc1.close();
        String[] array = arrayStr.split(",");
        int totalOfNumbers = 0;
        
        for (int index = 0; index < array.length; index++) {
            if(isInterger(array[index], n)) {
                totalOfNumbers++;
            }
        }
        return "Total de números inteiros = "+totalOfNumbers;
    }

    public static boolean isInterger(String value, int n) {
        if(!value.contains(".")){
            if(Integer.parseInt(value)<=n) {
                return true;
            } else {
                return false;
            }
        } else {
            String[] decimalValue = value.split("\\.");

            if(Integer.parseInt(decimalValue[0])<=n && Integer.parseInt(decimalValue[1])==0) {
                return true;
            } else {
                return false;
            }
        }
    }
}
